DOCS-4332: SCALE 22.12 Update VMWare Snapshot Articles (#1624)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md

---
title: "VMWare Snapshots Screen"
description: "This article provides information on the VMWare-Snapshot Add screen settings and functions."
weight: 55
tags:
- scalesnapshots
- scalevmware
---

> Use the **VMware Snapshot Integration** option on the **Data Protection > Periodic Snapshot Tasks** widget to create snapshots when TrueNAS SCALE is used as a VMWare datastore. 

Reviewer: ```suggestion
Use the **VMware Snapshot Integration** option on the **Data Protection > Periodic Snapshot Tasks** widget to create snapshots when you are using TrueNAS SCALE as a VMWare datastore. 
```
